plugins/modules/ilo_redfish_config.py

```diff
@@ -4,13 +4,14 @@
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 from __future__ import absolute_import, division, print_function
+
 __metaclass__ = type
 
 DOCUMENTATION = '''
 ---
 module: ilo_redfish_config
 short_description: Sets or updates configuration attributes on HPE iLO with Redfish OEM extensions
-version_added: 4.2.0
+version_added: 7.0.0
```

Comment:
This must not be changed. The module was added in community.general 4.2.0.

Reply:
Resolving this conversation, thanks!